model/bot.go

```diff
@@ -33,6 +33,20 @@ type Bot struct {
 	DeleteAt       int64  `json:"delete_at"`
 }
 
+func (b *Bot) Auditable() map[string]interface{} {
```

Comment:
I think it would be better if we auto-generate these methods. Keeping it up-to-date manually is fraught with danger, and it's very likely some dev will add a new field and forget to add that in this method.

It shouldn't be a big challenge technically. We just need to read the struct tags, and use them as the map keys and the struct fields as the values for the methods.

Not suggesting to do this right away, but something in the near future.